modules/nf-core/gem3/gem3mapper/main.nf

```diff
@@ -0,0 +1,46 @@
+process GEM3_GEM3MAPPER {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::gem3-mapper=3.6.1 bioconda::samtools=1.18"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-240a9c1936dd6a68f46aa198b2629b6734a18428:8fc2f8fd66e104a88e3eec4d8fec1665ee7f7278-0':
+        'biocontainers/mulled-v2-240a9c1936dd6a68f46aa198b2629b6734a18428:8fc2f8fd66e104a88e3eec4d8fec1665ee7f7278-0' }"
+
+    input:
+    tuple val(meta), path(index)
+    tuple val(meta2), path(fastq)
+
+    output:
+    tuple val(meta), path("*.bam"), emit: bam
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    gem-mapper -F 'SAM' -I $index -i $fastq -t $task.cpus | samtools view -bS -o ${prefix}.bam -
```

Comment:
Sometimes people may not want to sort, can you model this in a similar way to how we do it in bwa: https://github.com/nf-core/modules/blob/5709153c7abb192d48765693f8c5a642a35e100e/modules/nf-core/bwa/mem/main.nf#L35 ?

Reply:
Sure! I'll change it, you're right